modules/primary-cluster/main.tf

```diff
@@ -0,0 +1,74 @@
+locals {
+  instance_count = 3
```

Comment:
Can we re-add the comment for the limitation here? or add a new one around basically wanting a quorum-able control plane in addition to limitations with the internal service mode requiring no less than 3 primaries

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/58/commits/8d1e3ea8e7da180114b764d2cae5f03677833d6b.